sormas-backend/src/test/java/de/symeda/sormas/backend/contact/ContactServiceTest.java

```diff
@@ -111,16 +111,16 @@ public void testGetAllRelevantContacts() {
 		contact2.setOverwriteFollowUpUntil(false);
 		contact1.setFollowUpUntil(null);
 		contact2.setFollowUpUntil(null);
-		getContactFacade().saveContact(contact1);
-		getContactFacade().saveContact(contact2);
+		updateContact(contact1);
+		updateContact(contact2);
 
 		contacts = getContactService().getAllRelevantContacts(contactPersonEntity, contact1.getDisease(), referenceDate);
 		assertThat(contacts, hasSize(2));
 
 		contact1.setReportDateTime(DateHelper.subtractDays(referenceDate, ContactLogic.ALLOWED_CONTACT_DATE_OFFSET * 2));
 		contact2.setLastContactDate(DateHelper.subtractDays(referenceDate, ContactLogic.ALLOWED_CONTACT_DATE_OFFSET * 2));
-		getContactFacade().saveContact(contact1);
-		getContactFacade().saveContact(contact2);
+		updateContact(contact1);
+		updateContact(contact2);
 
 		contacts = getContactService().getAllRelevantContacts(contactPersonEntity, contact1.getDisease(), referenceDate);
 		assertThat(contacts, empty());
```

Comment:
@MartinWahnschaffeSymeda the change you suggested will cause a test failure in this line:

```
java.lang.AssertionError: 
Expected: an empty collection
     but: <[FirstName LASTNAME, FirstName LASTNAME]>
```